2023/assignments/fiesty-cushion/fiesty-cushion.md

```diff
@@ -0,0 +1,143 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>Braj Neupane</p>
+
+<p align="center">
+    <a href="https://youtu.be/dQw4w9WgXcQ" target="blank"
```

Comment:
@Fiesty-Cushion, kindly replace the current social media links with your authentic links.

Reply:
Don't have LinkedIn and Instagram yet so left that blank, rest should work.

Reply:
@suravshrestha, I have further updated the social bio links.